doofinder-for-woocommerce/includes/class-setup-wizard.php

```diff
@@ -1225,19 +1225,28 @@ private function set_layer_script($script)
             $options_suffix = '';
 
             if ($language_code !== $this->language->get_base_locale()) {
-                $options_suffix = Helpers::get_language_from_locale($language_code);
+                $language = str_replace("-", "_", $language_code);
+                $options_preffix = strtolower(explode("_", $language)[0]);
+                $options_suffix = strtolower(explode("_", $language)[1]);
```

Comment:
I would isolate explode("_", $language into a separated variable:

```suggestion
                $lang_parts      = explode( "_", $language );
                $options_preffix = strtolower( $lang_parts[0] );
                $options_suffix  = strtolower( $lang_parts[1] );
```